llvm/test/Transforms/SLPVectorizer/RISCV/ctpop.ll

```diff
@@ -1,8 +1,12 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
-; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv32 -mattr=+m,+v | FileCheck %s
-; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv64 -mattr=+m,+v | FileCheck %s
-; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv32 -mattr=+v,+zvbb | FileCheck %s
-; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv64 -mattr=+v,+zvbb | FileCheck %s
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv32 -mattr=+m,+v \
+; RUN: | FileCheck %s --check-prefixes=CHECK,RV32
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv64 -mattr=+m,+v \
+; RUN: | FileCheck %s --check-prefixes=CHECK,RV64
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv32 -mattr=+v,+zvbb \
+; RUN: | FileCheck %s --check-prefixes=CHECK,ZVBB
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv64 -mattr=+v,+zvbb \
+; RUN: | FileCheck %s --check-prefixes=CHECK,ZVBB32
```

Comment:
Should this just be --check-prefixes=CHECK,ZVBB?

Reply:
ZVBB64 was my intention: it has to be distinct from ZVBB or else `update_test_checks.py` reports a conflict. 

(Given that the output here in all cases except for RV64 is identical, I'm not sure what causes the conflict; I'd otherwise have assumed that two prefixes altogether, CHECK and RV64, suffice.)

Reply:
Yeah I find UTC often fails in the same way when there's too many combinations of check prefixes. Might just be easier to do `--check-prefixes=ZVBB` for the zvbb lines since they're interesting enough to look at on their own.  But this is a minor nit.